docker-compose.yml

```diff
@@ -23,6 +23,56 @@ services:
       DOCKER_INFLUXDB_INIT_ORG: eews
       DOCKER_INFLUXDB_INIT_BUCKET: eews
   
+  spark-master:
+    image: cluster-apache-spark:3.4.1
```

Comment:
any reason why not using the official or managed image like from bitnami?
https://hub.docker.com/r/apache/spark
https://hub.docker.com/r/bitnami/spark

Reply:
i am not aware of that before :(( thank you i'll revise asap